src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,68 @@
-export const Pagination = () => {};
+import { FC } from 'react';
+
+export interface IPagination {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: FC<IPagination> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  let totalPages = Math.ceil(total / perPage);
+
+  const changePage = (page: number) => {
+    if (page >= 1 && page <= totalPages) {
+      onPageChange(page);
+    }
+  };
+
+  const renderPageButtons = () => (
+    Array.from({ length: totalPages }, (_, index) => (
+      <li 
+      key={index+1}
+        className={`page-item ${currentPage===index+1?"active":''}`} 
+        onClick={() => changePage(index + 1)}
+        >
+        <a data-cy="pageLink" className="page-link" href={`#${index+1}`}>
```

Comment:
Consider using a more descriptive `href` value or a button element for the page links to improve accessibility and prevent default anchor behavior.